core/lib/env_config/src/contracts.rs

```diff
@@ -103,6 +104,7 @@ CONTRACTS_FRI_RECURSION_NODE_LEVEL_VK_HASH="0x5a3ef282b21e12fe1f4438e5bb158fc506
 CONTRACTS_FRI_RECURSION_LEAF_LEVEL_VK_HASH="0x72167c43a46cf38875b267d67716edc4563861364a3c03ab7aee73498421e828"
 CONTRACTS_PROVER_AT_GENESIS="fri"
 CONTRACTS_SNARK_WRAPPER_VK_HASH="0x4be443afd605a782b6e56d199df2460a025c81b3dea144e135bece83612563f2"
+CONTRACTS_BASE_TOKEN_ADDR=0x0000000000000000000000000000000000000001
```

Comment:
You set `base_token_addr: addr("0x0000000000000000000000000000000000000000"),` above, so probably this test would fail.

Reply:
Fixed